models/var_groups.yaml

```diff
@@ -159,23 +159,23 @@ var_groups:
       - entity_var:
           name: avg_session_length_in_sec_overall
           from: models/rsTracksUnionPages
-          select: avg({{ macro_datediff('second, {{rsTracksUnionPages.Var("session_start_time")}}, {{rsTracksUnionPages.Var("session_end_time")}}') }})
+          select: "avg({{ macro_datediff_with_start_end_and_granularity('second', '{{rsTracksUnionPages.Var(\"session_start_time\")}}', '{{rsTracksUnionPages.Var(\"session_end_time\")}}') }})"
```

Comment:
nit: The syntax of this `select` can be same as the other two (like drop the outermost quotes)